students/nico/index.html

```diff
@@ -5,44 +5,62 @@
   <title>Nico's Page</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Major+Mono+Display|Raleway:400,700" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">  
   <meta property="og:title" content="myWebPage">
   <meta property="og:type" content="website">
   <meta property="og:url" content="file:///Users/nicolacaze/Desktop/Ironhack/m1/w1/myPersoPage/index.html">
   <meta property="og:image" content="https://upload.wikimedia.org/wikipedia/en/thumb/b/b4/Donald_Duck.svg/220px-Donald_Duck.svg.png">
 </head>
 
 <body>
-  <header>
-    <h1>Nicolas Lacaze</h1>
-    <h2>Full stack developer</h2>
+  <header class="header-container">
+    <a href="https://tawebbcn.github.io/cohort0191"><img class="header-container-item" src="img/left-arrow.png" alt="back-arrow"></a>
+    <nav class="header-container-item">
+      <ul class="nav-list-container">
+        <li class="nav-list-items"><a href="#about"/>Me</a></li>
+        <li class="nav-list-items"><a href="#portfolio">Portfolio</a></li>
+        <li class="nav-list-items"><a href="#contact">Contact</a></li>
```

Comment:
I don't think that you need a class for everything, sometimes remember that you can use the element to style it. For example you can use in the css the class of the navbar and then refer to the li.

Reply:
I made a few changes as you suggested.